recipe/meta.yaml

```diff
@@ -1,15 +1,22 @@
+{% set name = "catboost" %}
+{% set version = "1.2" %}
+
 package:
-  name: catboost
-  version: "1.0.6"
+  name: {{ name|lower }}
+  version: {{ version }}
 
 build:
-  number: 1
+  number: 0
   #  Yandex supplies whl files on PyPI for:
-  # - Linux: Python 3.6, 3.7, 3.8, 3.9, 3.10
-  # - OS X: Python 3.6, 3.7, 3.8, 3.9, 3.10
-  # - Win-64: Python 3.6, 3.7, 3.8, 3.9, 3.10
-  skip: True  # [win32 or python_impl == 'pypy']
-  skip: True  # [linux and not x86_64]
+  # - Linux: Python 3.8, 3.9, 3.10, 3.11
+  # - OS X: Python 3.8, 3.9, 3.10, 3.11
+  # - Win-64: Python 3.8, 3.9, 3.10, 3.11
+  # There are issues with python 3.8 on macOS 11.x: https://github.com/conda-forge/python-feedstock/issues/445
+  #  so Conda has issues with tags compatibility with platform tags 'macosx_11_0_...'
+  #  so disable this configuration for now
+  skip: True  # [(osx and x86_64) and py==38]
+  # TODO: support Linux on ppc64le: https://github.com/catboost/catboost/issues/2145 and s390x
+  skip: True  # [ppc64le or s390x]
```

Comment:
why are we skipping s390x?

Reply:
Missing wheels for this platform:
`ERROR: catboost-1.2-cp38-cp38-manylinux2014_.whl is not a supported wheel on this platform.`

Reply:
is it the same for all py versions?

Reply:
No, for example, catboost-1.2-cp311-cp311-**manylinux2014_aarch64**.whl is for `linux-aarch64`.
There are missing wheels for s390x https://pypi.org/project/catboost/1.2/#files

Reply:
Ah, there is no value for s390x https://github.com/AnacondaRecipes/catboost-feedstock/pull/4/files#diff-d7075654874cb08007a21aaab3ecd4b3453a9087e7505d034d548b8938b599bcR41-R44